RadixWallet/Features/AccountDetailsFeature/Coordinator/AccountDetails+View.swift

```diff
@@ -20,8 +20,8 @@ extension AccountDetails {
 		let accountAddress: AccountAddress
 		let appearanceID: Profile.Network.Account.AppearanceID
 		let displayName: String
-		let needToImportMnemonicForThisAccount: Bool
-		let needToBackupMnemonicForThisAccount: Bool
+		let importMnemonicNeeded: Bool
```

Comment:
Minor: I guess both cannot be true at the same time, could be an enum.

Reply:
Mm I realized that too last night, earlier it did bit work but we have changed some prerequisites now